src/pl/devsmentoring/java/Module5/Exercise20.java

```diff
@@ -0,0 +1,50 @@
+package pl.devsmentoring.java.Module5;
+
+import java.util.*;
+
+public class Exercise20 {
+    public static void main(String[] args) {
+
+        List<String> colourList = Arrays.asList("zielony", "czerwony", "niebieski", "czarny",
+                "fioletowy", "granatowy", "niebieski",
+                "czarny", "czarny", "zielony", "cytrynowy", "granatowy",
+                "niebieski", "indygo", "zielony", "czerwony");
+
+        System.out.println("Original length of our colour list was: " + colourList.size());
+
+
+        Map<String, Integer> colorCounts = new HashMap<>();
+        for (String color : colourList) {
+            colorCounts.putIfAbsent(color, 0);
+            colorCounts.put(color, colorCounts.get(color) + 1);
+        }
+
+        for (Map.Entry<String, Integer> entry : colorCounts.entrySet()) {
+            System.out.println("Colour numbers of occurrences: " + entry.getKey() + " - " + entry.getValue());
+        }
+
+
+        Set<String> colourAsValues = new HashSet<>(colourList);
```

Comment:
już nie potrzebujesz deklarować setów. W zadanku raczej chodziło o to żebyś spróbował do początkowej listy dodać jakiś kolor i zobaczyć jak zmieni się program